src/Form/input-classifiers.test.js

```diff
@@ -196,11 +208,12 @@ describe.each(testCases)('Test $html fields', (testCase) => {
         if (bad.length !== expectedFailures.length) {
             for (let score of bad) {
                 console.log(
-                    'manualType:   ' + JSON.stringify(score.manualScore),
+                    'file:         ' + html,
```

Comment:
I've added the file name so it's easier to track down when reviewing the console. The rest is just to unify the formats.